src/main/java/lotto/controller/LottoController.java

```diff
@@ -14,17 +17,37 @@
     public void start() {
         OutputView outputView = new OutputView();
 
-        Ticket ticket = InputView.buyTicket();
-        Lottos lottoRepository = TicketDealer.getLottosOf(new RandomLottoGenerator(), ticket.getBuyCount());
+        Money money = getMoney();
+        PurchaseTicket ticket = new PurchaseTicket(money, getManualCount());
+        Lottos lottos = ticket.purchase(getManualLottoNumbers(ticket));
```

Comment:
```suggestion
        PurchaseTicket ticket = new PurchaseTicket(money, manualLottoNumbers());
        Lottos lottos = ticket.purchase();
```
`PurchaseTicket`에 로또번호만 넘기고 LottoSupplier을 이용해 수동로또 생성까지 해보는 것은 어떨까요?